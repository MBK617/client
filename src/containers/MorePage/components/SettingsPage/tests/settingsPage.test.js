import React from 'react';
import { render, cleanup } from '@testing-library/react-native';

import SettingsPage from '..';

describe('<SettingsPage />', () => {
  let rendered;
  beforeEach(() => {
    rendered = render(<SettingsPage />);
  })

  afterEach(cleanup);

  it('should render correctly', () => {
    const tree = rendered.toJSON();
    expect(tree).toMatchSnapshot();
  });
});