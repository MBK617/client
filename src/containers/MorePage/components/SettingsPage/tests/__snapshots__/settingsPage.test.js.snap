// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SettingsPage /> should render correctly 1`] = `
<View>
  <Text>
    Settings
  </Text>
</View>
`;
